python/ola/PidStore.py

```diff
@@ -120,8 +122,8 @@ def GetRequest(self, command_class):
     return self._requests.get(command_class)
 
   def GetRequestField(self, command_class, field_name):
-    fields = filter(lambda field: field.name == field_name,
```

Comment:
The filter is now as part of the list comprehension, the ```if f.name == field_name``` part.

The "why" of that change is explained in the commit message.

Note that I split the line because it was a bit long, but it could be written directly as:
```python
fields = [f for f in self.GetRequest(command_class).GetAtoms() if f.name == field_name]
```